src/tests/ftest/recovery/ddb.py

```diff
@@ -23,10 +23,15 @@ class DdbTest(TestWithServers):
     def __init__(self, *args, **kwargs):
         """Initialize a DdbTest object."""
         super().__init__(*args, **kwargs)
+        # how many objects and keys to insert/expect
+        self.object_count = 5
+        self.dkey_count = 2
+        self.akey_count = 1
         # Generate random keys and data to insert into the object.
         self.random_dkey = get_random_string(10)
         self.random_akey = get_random_string(10)
         self.random_data = get_random_string(10)
+        self.standalone = False  # enables running test in a "stand alone" mode
```

Comment:
Is this for future work? It seems always `False`

Reply:
This was so I could iterate over the test on an existing vos file without test needing to create it each time, So in a temporary script I could do something like:  
```
test = DdbTest(methodName='test_recovery_ddb_ls')
test.standalone = True
(...)
test.test_recovery_ddb_ls()
```

I'm open to suggestions or can just remove it all together.

Reply:
That's an interesting use case - one which we don't consider in the framework. The problem with using `self.standalone` like this is that it won't get covered by most test changes, so will eventually break. I.e. I don't imagine anyone is going to test that when making changes. I'd be in favor removing it, if not too inconvenient.
To support your use case while not introducing unused code paths, I think the simplest approach would be to break the test method into helper functions such that you can call the helper functions as needed. But that's a pretty big re-org of this test.

Reply:
I'll remove "standalone". Thanks.